sustain-example.md

```diff
@@ -0,0 +1,32 @@
+This markdown file is just as important as a README. It ensures those using this open source software that there are funds coming in as well as a solid back-up plan in case of an debilitating illness, accident, or untimely death.
+
+## Funding
+We receive funding from the following platform(s):
+
+* https://www.patreon.com/sustainmd-example
+* https://opencollective.com/sustainmd-example
+
+If you or your organization gets value out of this project, please consider donating.
+
+## Bus factor
+The follow individuals have agreed to take over in case of an untimely death or debilitating illness and/or accident.
```

Comment:
In addition to who is willing to take over, it would be helpful to know who is currently responsible.

Current Maintainers are:
- Justin
  - contact info
- Georg
  - contact info

Reply:
my only fear is people will accidentally put that in their repo and then we get flooded with irrelevant support questions. If you think I am catastrophizing, LMK and I will most likely change my mind.

Reply:
I agree that we should not put our own names in the example.

My problem is with the wording "take over in case of an untimely death" which implies that they are currently not maintainers and we don't know who has to die for them to take over.

Maybe a better wording could be:
The following individuals are maintainers and have the skills, know-how, and access to continue the project in case any one of them gets hit by a bus:

Reply:
That sounds better. Want to make a commit or I can do it? Your call.

Reply:
Committed